stan/math/rev/mat/fun/divide.hpp

```diff
@@ -9,18 +9,165 @@
 namespace stan {
 namespace math {
 
+namespace internal {
+template <int R, int C>
+class matrix_scalar_divide_vari_dv : public vari {
```

Comment:
[optional]

I believe the convention we've used elsewhere is to have the `vari` extensions end in `vari`, so I'd suggst `_dv_vari` for the name and similarly elsewhere.

Reply:
You're right, changed for consistency.